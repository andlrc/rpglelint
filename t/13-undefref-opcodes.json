[
   {
      "column":3,
      "file":"t/13-undefref-opcodes.rpgle",
      "line":4,
      "message":"'a' undeclared",
      "severity":"warning"
   },
   {
      "column":3,
      "file":"t/13-undefref-opcodes.rpgle",
      "line":6,
      "message":"'b' undeclared",
      "severity":"warning"
   }
]
