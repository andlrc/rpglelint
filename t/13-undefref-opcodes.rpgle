// rpglelint: -Wundefined-reference

if *INLR > *ON;
  a = 1;
else;
  b = 2;
endif;
