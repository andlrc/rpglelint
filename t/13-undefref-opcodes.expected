t/13-undefref-opcodes.rpgle:4:3: warning: 'a' undeclared [-Wundefined-reference]
   a = 1;
   ^
t/13-undefref-opcodes.rpgle:6:3: warning: 'b' undeclared [-Wundefined-reference]
   b = 2;
   ^
